### ISSUE TITLE
Events on the parent element are not triggered on empty regions of the chart

### ISSUE THREAD

**Issue by jadfls** [FormidableLabs/victory-native#603]
# Bugs and Questions

### Checklist

* [x] This is a `victory-native` specific issue. (Issues that _also_ appear in `victory` should be opened [here](https://github.com/FormidableLabs/victory/issues/new))

* [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

* [x] I am using the latest version of `victory-native`

* [x] I have checked to make sure that my versions of `react-native` and `react-native-svg` are compatible with this version of `victory-native`. [Read about version requirements](https://github.com/FormidableLabs/victory-native/blob/main/README.md#peer-dependencies-and-version-requirements)

* [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

VictoryVoronoiContainer does not always find the closest data point in React-Native.
Although it does well in Web version. See Reproduction.

### Reproduction

When using following code in Web version, you can see how the closest point is always highlighted when you move around with the mouse. Even if you place the mouse in the most bottom-left corner, or top-right corner, where it is far away from data points, it will choose the closest.

```
<VictoryChart domainPadding={{ y: 10 }}
  containerComponent={
    <VictoryVoronoiContainer
    voronoiDimension="x"
      labels={({ datum }) => `${round(datum.x, 2)}, ${round(datum.y, 2)}`}
    />
  }
>
  <VictoryLine
    y={(datum) => Math.sin(2 * Math.PI * datum.x)}
  />
</VictoryChart>
```

Now, when you translate that to React Native, e.g. following code. You can see that it does not behave the same. When I touch around over the line, the data points are shown correctly. But when I move far away, in the most bottom-left corner, or top-right corner, it will not show anything. Tested on iOS and Android.

```
import {VictoryChart, VictoryLine, VictoryVoronoiContainer} from "victory-native";
import React from "react";
import {View} from "react-native";


export default class CustomDataComponent extends React.Component {

  render () {
    return (
        <View style={{"flex":1}}>
        <VictoryChart domainPadding={{ y: 10 }}
            containerComponent={
                <VictoryVoronoiContainer
                voronoiDimension="x"
                labels={({ datum }) => `${Math.round(datum.x, 2)}, ${Math.round(datum.y, 2)}`}
                />
            }
            >
        <VictoryLine
            y={(datum) => Math.sin(2 * Math.PI * datum.x)}
        />
        </VictoryChart>
        </View>

    );
  }
}
```

**Comment by boygirl** [comment]
@jadfls I haven't been able to reproduce this issue. What version of victory-native are you using?

**Comment by jadfls** [comment]
I am using
    victory-native: 35.0.1
    react-native-svg: 12.1.0,
    react-native: 0.63.2

Tried and verified again on my iOS device

**Comment by jadfls** [comment]
I added few gifs to visualize the issue: https://imgur.com/a/Jpw5FD7

**Comment by boygirl** [comment]
@jadfls Thank you for the gifs, that's super helpful. I'll do some digging today and see what I can come up with.

**Comment by boygirl** [comment]
After a little more investigation, it looks like the issue is not that the voronoi logic is incorrectly associating points, but that the events that should be triggered by clicking on the parent element are not being triggered except where other chart elements exist. It's a lot easier to see with bar charts than line charts

![native-voronoi-issue](https://user-images.githubusercontent.com/3719995/98734321-a528b480-2356-11eb-89a1-3769bf1e12f8.gif)

**Comment by boygirl** [comment]
@jadfls Thanks again for the gifs, they were instrumental in figuring out what was going on. Unfortunately the root issue is a bug in `react-native-svg` which I documented here: https://github.com/react-native-svg/react-native-svg/issues/1488, but I was able to make a temporary fix. I released it as `victory-native@35.3.1`. I hope it works well for you
